lib/zspec/formatter.rb

```diff
@@ -33,7 +33,13 @@ def close(_notification)
         @stdout.string
       )
       @tracker.track_runtime(@message, @duration)
-      @tracker.track_failures(@output_hash[:failures]) if @failed
+      if @failed
+        unless @output_hash[:failures].empty?
```

Comment:
thoughts on flipping this from an `unless` to an `if`? always makes it easier to read that way, imo

```suggestion
        if @output_hash[:failures].any?
```